languages/go/README.md

```diff
@@ -24,10 +24,10 @@ This area will contain everything needed to launch the Go track, including:
 Before we publicize requesting contribution for this language, the following steps should be done.
 
 - [x] [Convert existing files to new repository structure](../../docs/maintainers/repository-structure.md)
-- [ ] Have a kick-off discussion between track maintainers
-- [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
+- [x] Have a kick-off discussion between track maintainers
+- [x] Fill out the [maintainers.md](./maintainers.md) file ([Go](maintainers.md))
```

Comment:
Would it perhaps make sense to remove the bit between parentheses too?

```suggestion
- [x] Fill out the [maintainers.md](./maintainers.md) file
```

Reply:
Right, didn't see it's there already.